lua/astrocommunity/editing-support/true-zen-nvim/true-zen-nvim.lua

```diff
@@ -0,0 +1,34 @@
+local utils = require "astronvim.utils"
+return {
+  "Pocco81/true-zen.nvim",
+  opts = {
+    integrations = {
+      tmux = os.getenv "TMUX" ~= nil, -- hide tmux status bar in (minimalist, ataraxis)
+      twilight = utils.is_available "twilight.nvim", -- enable twilight (ataraxis)
```

Comment:
I don't think this will work necessarily because `is_available` relies pretty heavily on lazy being done loading. I think this could be resolved by using the function notation for setting `opts`